src/Microsoft.AspNetCore.SignalR/IHubClients.cs

```diff
@@ -1,12 +1,16 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using System.Collections.Generic;
+
 namespace Microsoft.AspNetCore.SignalR
 {
     public interface IHubClients
     {
         IClientProxy All { get; }
 
+        IClientProxy AllExcept(List<string> excludedIds);
```

Comment:
I don't think having `params` on a public interface API is a great plan actually... I'd rather use `IEnumerable` or `IReadOnlyList` and have a `params` variant implemented in an extension method. `params` forces you to allocate an array, if you don't already have one (until we get `params IEnumerable` someday...)